CHANGELOG.rst

```diff
@@ -6,6 +6,80 @@
 Synapse Changelog
 *****************
 
+v2.149.0 - 2023-09-14
+=====================
+
+Model Changes
+-------------
+- Update to the ``it``, ``meta``, and ``org`` models.
+  (`#3338 <https://github.com/vertexproject/synapse/pull/3338>`_)
+
+  **New Properties**
+
+  ``taxonomoy``
+    The interface had the following property added to it:
+
+    ``description``
+      'A definition of the taxonomy entry.
+
+  ``inet:email:message``
+    The form had the following property added to it:
+
+    ``cc``
+      Email addresses parsed from the "cc" header.
+
+  ``meta:source``
+    The form had the following property added to it:
+
+    ``url``
+      A URL which documents the meta source.
+
+  ``ou:campaign``
+    The form had the following property added to it:
+
+    ``timeline``
+      A timeline relevent events in the campaign.
```

Comment:
Typo "relevant", also I think a word is missing, maybe "A timeline of"

Reply:
We'll need to update the docstring here https://github.com/vertexproject/synapse/pull/3338/files#diff-a5a2cc80b286f378f0abaa9aef393577e904863d2b59319db934971b5a0e9dacR456